mirror/github/download.py

```diff
@@ -0,0 +1,75 @@
+import httpx
```

Comment:
@Andrei-Dolgolev : Give this file a different name. `download.py` is very generic and I don't understand what is being downloaded. This is specifically something like `popular_repos.py`, right?